cardano_clusterlib/clusterlib_klass.py

```diff
@@ -2514,6 +2537,7 @@ def send_tx(
             tx_name=tx_name,
             txins=txins,
             txouts=txouts,
+            readonly_reference_txins=readonly_reference_txins,
```

Comment:
Also missing in `self.calculate_tx_fee` call above.

Reply:
Updated. There we don't have the new fields related to collaterals, no need?

Reply:
Not sure if that new collateral stuff increase tx size / affect tx fee, we'll need to test that.